ios/RCTMapboxGL/RCTMapboxGL.h

```diff
@@ -44,6 +43,7 @@
 - (double)zoomLevel;
 - (double)direction;
 - (CLLocationCoordinate2D)centerCoordinate;
+- (void)setContentInset:(UIEdgeInsets)inset;
```

Comment:
Any reason for not exposing a `-contentInset` getter? (You could declare both at one time with `@property UIEdgeInsets inset;`.